java11/src/test/resources/verifyActivityResponse.json

```diff
@@ -1,3 +1,6 @@
 {
-  "emails": 0.61
+  "emails" : 0.82,
+  "online" : 0.91,
+  "social" : 0.32,
+  "employment" : 0.78
```

Comment:
This is mocked for test to pass. I do wonder if we should have a test for this library with live api calls in the future.